scripts/create-latest-svc.sh

```diff
@@ -0,0 +1,130 @@
+#/bin/bash
+
+set -e
+
+#
+# Downloads latest releases (not pre-release) runner
+# Configures as a service
+#
+# Examples:
+# RUNNER_CFG_PAT=<yourPAT> ./create-latest-svc.sh myuser/myrepo
+# RUNNER_CFG_PAT=<yourPAT> ./create-latest-svc.sh myorg
+#
+# Usage:
+#     export RUNNER_CFG_PAT=<yourPAT>
+#     ./create-latest-svc scope [name] [user]
+#
+#      scope required  repo (:owner/:repo) or org (:organization)
+#      name  optional  defaults to hostname
+#      user  optional  user svc will run as. defaults to current
+# 
+# Notes:
+# PATS over envvars are more secure
+# Should be used on VMs and not containers
+# Works on OSX and Linux 
+# Assumes x64 arch
+#
+
+runner_scope=${1}
+runner_name=${2:-$(hostname)}
```

Comment:
If I'm spinning up a lot of VMs to run this script and create runners, are we guaranteed to have a unique hostname for each one?

Reply:
That's why we take it as arg2.  But the vast majority of the time, hostname being the default is correct.

Reply:
I missed that, was looking at the Examples in this script file and didn't look at usage.

Should we have flags for those inputs? I.e. what if I want to specify User but not Name

Also thinking ahead to labels inputs, relying on the order of the arguments might not hold up

Reply:
It's simple and customary for bash scripting.  I thought about the user issue - it's a real edge and in that case you put $(hostname) in front of it.  If we did flags I think the sample script would get bogged down with a bunch of parsing logic and folks won't be able to see the forest through the weeds of the script.  That is, unless someone knows of a trivial (works everywhere) cmd line parsing bash technique ...